test/integration/import_and_export/conftest.py

```diff
@@ -0,0 +1,186 @@
+import copy
+import csv
+from dataclasses import dataclass
+from datetime import datetime
+from inspect import cleandoc
+from io import StringIO
+from pathlib import Path
+from typing import Optional
+
+import pytest
+
+ALL_COLUMNS = [
+    "FIRST_NAME",
+    "LAST_NAME",
+    "REGISTER_DT",
+    "LAST_VISIT_TS",
+    "IS_GRADUATING",
+    "AGE",
+    "SCORE",
+]
+DATETIME_STR_FORMAT = "%Y-%m-%d %H:%M:%S.%f"
+TABLE_NAME = "USER_SCORES"
+
+
+@pytest.fixture
+def table_name():
+    return TABLE_NAME
+
+
+@pytest.fixture(scope="session", autouse=True)
+def faker_seed():
+    return 12345
+
+
+@pytest.fixture(autouse=True)
```

Comment:
Not a fan of autouse, if there is no specific reason for it. Does each test really need the table? If there is a specific reason, please add a comment.